src/script.js

```diff
@@ -1,6 +1,10 @@
 var page = require('webpage').create();
 var system = require('system');
 
+
+phantom.setProxy(system.args[2], system.args[3]) //setProxy('ip','port')
```

Comment:
Do we need to check if `system.args[2]` and `system.args[3]` exist before calling setProxy?